apps/src/code-studio/pd/application/PrivacyDialog.jsx

```diff
@@ -47,14 +49,16 @@ export default class PrivacyDialog extends React.Component {
             </a>
             .
           </p>
-          <p style={STYLE.bodyText}>
-            Teachers may be required to get principal approval for their
-            application to the Professional Learning Program. As part of this
-            process principals may opt in to let the College Board share
-            de-identified and aggregated Computer Science AP scores with
-            Code.org to help us improve the program and curriculum. AP test
-            scores will not be shared with Regional Partners.
-          </p>
+          {!this.props.principalApproval && (
```

Comment:
this is small, but I think it's a little nicer to pass in a bool for `teacherApplication` and showing this when it's true instead of using `principalApproval` and doing something when it's false. Also because conceptually this paragraph should be shown specifically when it _is_ the teacher application, not when it's not the principal approval form (if this dialog were used in more places)

Reply:
You're absolutely right.  Good call.  (I had erred on the side of "backwards compatibility" but it's only used in two places, and the semantics of being teacher-specific are spot-on.)